test/src/Main.purs

```diff
@@ -123,8 +208,15 @@ timeTest = do
   log $ "\nUNFORMATDATETIME"
   DT.traceAnyA $ FDT.unformatDateTime "YYYY-DD-MM SSS" "3456-09-10 333"
 
-
-main ∷ forall e. Eff (console ∷ CONSOLE | e) Unit
-main = do
-  timeTest
-  numeral
+main :: forall e.
+  Eff ( process :: PROCESS, exception :: EXCEPTION, console :: CONSOLE | e)
+    (Canceler ( process :: PROCESS, exception :: EXCEPTION, console :: CONSOLE | e))
+main = execTests tests true
+  where
+  tests = do
+    log "Testing time functions..."
+    timeTest
+    passed <- get
+    when (passed /= true) (throwError (error "Tests did not pass."))
+    --numeralTests
+    --formattingTests
```

Comment:
That's fine :)  I forgot to mention those.  I think we could probably uncomment them if you think it's valuable, but I wasn't exactly sure what those functions were testing so I didn't know how valuable they were.  They seemed more like output that was used for visual feedback when the previous developer was working on it.  They should probably be migrated to more explicit assertions that cause the process to exit with 1 if they fail, but I didn't have time to do that.